src/Controls/samples/Controls.Sample.UITests/Issues/Issue18740.xaml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue18740">
+    <StackLayout>
+        <Label 
+            AutomationId="WaitForStubControl"
+            Text="1. Make sure keyboard starts out closed."/>
+        <Label 
+            Text="2. Focus the Entry."/>
+        <Label 
+            Text="3. Verify that the virtual keyboard appears."/>
+        <Entry 
```

Comment:
The original device test validated against the Editory/Entry/SearchBar so we'll need to validate against all three of those types to replace the device test.

Reply:
Good point. Added Editor/Entry/SearchBar to the test.